subprojects/architecture-test/src/changes/archunit_store/public_api_methods_do_not_reference_internal_types_as_parameters.txt

```diff
@@ -0,0 +1,380 @@
+Method <org.gradle.util.ClosureBackedAction.execute(java.lang.Object, groovy.lang.Closure)> has arguments/return type groovy.lang.Closure that is not Gradle public API or primitive or built-in JDK classes or Kotlin classes in (ClosureBackedAction.java:0)
+Method <org.gradle.util.ClosureBackedAction.getClosure()> has arguments/return type groovy.lang.Closure that is not Gradle public API or primitive or built-in JDK classes or Kotlin classes in (ClosureBackedAction.java:0)
+Method <org.gradle.util.ClosureBackedAction.of(groovy.lang.Closure)> has arguments/return type groovy.lang.Closure that is not Gradle public API or primitive or built-in JDK classes or Kotlin classes in (ClosureBackedAction.java:0)
+Method <org.gradle.util.CollectionUtils.partition(java.lang.Iterable, org.gradle.api.specs.Spec)> has arguments/return type org.gradle.internal.Pair that is not Gradle public API or primitive or built-in JDK classes or Kotlin classes in (CollectionUtils.java:0)
+Method <org.gradle.util.Configurable.configure(groovy.lang.Closure)> has arguments/return type groovy.lang.Closure that is not Gradle public API or primitive or built-in JDK classes or Kotlin classes in (Configurable.java:0)
+Method <org.gradle.util.ConfigureUtil$WrappedConfigureAction.getConfigureClosure()> has arguments/return type groovy.lang.Closure that is not Gradle public API or primitive or built-in JDK classes or Kotlin classes in (ConfigureUtil.java:0)
+Method <org.gradle.util.ConfigureUtil.configure(groovy.lang.Closure, java.lang.Object)> has arguments/return type groovy.lang.Closure that is not Gradle public API or primitive or built-in JDK classes or Kotlin classes in (ConfigureUtil.java:0)
+Method <org.gradle.util.ConfigureUtil.configureSelf(groovy.lang.Closure, java.lang.Object)> has arguments/return type groovy.lang.Closure that is not Gradle public API or primitive or built-in JDK classes or Kotlin classes in (ConfigureUtil.java:0)
+Method <org.gradle.util.ConfigureUtil.configureSelf(groovy.lang.Closure, java.lang.Object, org.gradle.internal.metaobject.ConfigureDelegate)> has arguments/return type groovy.lang.Closure, org.gradle.internal.metaobject.ConfigureDelegate that are not Gradle public API or primitive or built-in JDK classes or Kotlin classes in (ConfigureUtil.java:0)
+Method <org.gradle.util.ConfigureUtil.configureUsing(groovy.lang.Closure)> has arguments/return type groovy.lang.Closure that is not Gradle public API or primitive or built-in JDK classes or Kotlin classes in (ConfigureUtil.java:0)
+Method <org.gradle.util.GFileUtils.checksum(java.io.File, java.util.zip.Checksum)> has arguments/return type java.util.zip.Checksum that is not Gradle public API or primitive or built-in JDK classes or Kotlin classes in (GFileUtils.java:0)
```

Comment:
@wolfs Same thing here.  This is a JDK type.

Reply:
Yeah, but we don't allow all JDK types. It shows now up as new since you'd need to point to the right freeze file as I said in my other comment.